Ownership/TC - LTC/London TC NE.txt

```diff
@@ -1,32 +1,24 @@
 SECTOR:London TC NE:0:15500
 OWNER:TCNE:TCN:TC:TCE:LE:LC:LNC:LWC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:TCNE:TCN:TC:TCE:LE:LC:LNC:LWC:O:LM:LSC:L
```

Comment:
I have rearranged the ALTOWNER from smallest sector to biggest, hence it looks like there are more changes here than there actually are.